app/models/ArticleModel.js

```diff
@@ -0,0 +1,47 @@
+(function(exports) {
+
+  var articleCounter = 0;
+
+  exports.ArticleModel = function(headline, url, body, main) {
+    var _id = articleCounter;
+    articleCounter++;
+
+    var storyBody = document.createElement("div");
```

Comment:
The model feels like the wrong place to be building the DOM. Your model should be responsible for 'what' (i.e. the information to be injected into the view) and your view should know 'how' (i.e. how to display that information).

I note that you've commented on this in your self review, so a bit more info on how that might look:

```
function render(model) {
   var html = "<div class='some-class'><h1 class='title'>{title}</h1></div>"
   var htmlWithModelData = html.replace('{title}', model.title)
   document.getElementById("app").innerHTML= htmlWithModelData;
}
```
EDIT: just seen that you've adopted more of this approach in your view